meta-resin-common/recipes-containers/balena/balena/balena-healthcheck

```diff
@@ -7,3 +7,6 @@ balena info > /dev/null 2>&1
 
 # Check that we can read containers from disk
 balena ps > /dev/null
+
+# Check that we can start a new container
+balena run --rm hello-world > /dev/null
```

Comment:
Won't this fail if the hello-world image isn't available and there's no connectivity?

Reply:
I've updated the PR so that the hello-world image is pre-downloaded at build time. and pulled during hup.